engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -51,6 +52,15 @@
     );
 
     /**
+     * Sometimes, a tracking system (i.e. for room-scale VR) provides a pose for the player hand position. In this
+     * case, special handling is necessary, and this accessor gives a way to check.
+     * @return true if this class is receiving tracking updates, false if not.
+     */
+    public boolean isTracked() {
+        return trackingDataReceived;
+    }
```

Comment:
I've tried to skirt the line between specificity and generality here. If I leave my comments too vague (is tracked? How?), then reading the javadocs doesn't make any sense.

The way I see it, a "tool mount point" is almost always a hand. so I used the two interchangeably in this case. However, I've adjusted the Javadoc here to only reference "tool mount point", because I can see how referring to a hand here could be confusing.

The suggestion I made is that I would make a class inside of the new VR subsystem that implements `ControllerListener`, holds reference to the mount point, and edits the quaternion on the mount point. We could then remove the implementation of `ControllerListener` here. That seems to jive more with the "systems modify components" architecture.This is still on my road map.

I still see this method as being necessary after I perform the above change. Regardless of what is doing the updating of the location info on the component, the special handling needed by other components (e.g. `FirstPersonClientSystem`, `PlayerTargetSystem`) hinges around the quaternion / position being updated dynamically rather than needing special handling as dumb objects. More specifically, without tracking information, animations are needed when actions are triggered, and there is no information that contributes to targeting. My language here seems general enough to reflect that, and also includes examples for clarification.